common/chroot-style/proot.sh

```diff
@@ -18,7 +18,7 @@ if [ -z "$MASTERDIR" -o -z "$DISTDIR" ]; then
 fi
 
 # proot does not properly return the resultcode. Workaround this
-RESULT=$(mktemp /tmp/proot_result.XXXXXXXXXX)
+RESULT=$(mktemp || exit 1)
```

Comment:
the `exit` here only exits the subshell that calls `mktemp`, you need to check the return value of the assignment statement if you want to catch errors from it

```sh
# exit with 1 if mktemp fails (this is probably what you meant)
RESULT=$(mktemp) || exit 1
```

to do this with `local`, you have to [declare and assign separately to avoid masking return values](https://github.com/koalaman/shellcheck/wiki/SC2155)

Reply:
You are absolutely right! thanks, my sed line failed.

Reply:
You are absolutely right! thanks, my sed line failed.

Reply:
You are absolutely right! thanks, my sed line failed.